components/encoder/encoder.go

```diff
@@ -82,14 +84,21 @@ func (t PositionType) String() string {
 //
 //	err = myEncoder.ResetPosition(context.Background(), nil)
 //
+// For more information, see the [ResetPosition method docs].
+//
 // Properties example:
 //
 //	myEncoder, err := encoder.FromRobot(machine, "my_encoder")
 //
 //	// Get whether the encoder returns position in ticks or degrees.
 //	properties, err := myEncoder.Properties(context.Background(), nil)
 //
-// [encoder component docs]: https://docs.viam.com/components/encoder/
+// For more information, see the [Properties methods docs].
+//
+// [encoder component docs]: https://docs.viam.com/operate/reference/components/encoder/
```

Comment:
```suggestion
// [encoder component docs]: https://docs.viam.com/dev/reference/apis/components/encoder/
```
Actually, let's just link to the API for all components since the plan is to consolidate this page with the API page soon anyway, and then the links will already be up to date.